aten/src/ATen/native/CPUBlas.h

```diff
@@ -88,6 +88,15 @@ void gemm(
     float beta,
     at::Half *c, int64_t ldc);
 
+void gemm(
```

Comment:
I understand you mainly want this for flashattention, but this seems like a general purpose matmul CPU kernel for float16? Would it make sense to also expose this to torch.mm for CPU and extend test coverage there too? That way you have more test coverage for flash attention too.

Reply:
Thanks for your comments. mkl_gemm_f16f16f32 is only added for flash attention. It is same as https://github.com/pytorch/pytorch/pull/107196. In the attention kernel, q @ k.T = attn, q and k will be given as float16 and attn is float32.